po/es.po

```diff
@@ -0,0 +1,245 @@
+msgid "Alpha-Value-Position"
+msgstr "Valor-Alfa-Posición"
+
+#: data/resources/ui/preferences.ui:17
+msgid "Where the Alphavalue is positioned in the Hexstring"
+msgstr "Dónde se sitúa el valor alfabético en la cadena hexadecimal"
+
+#: data/resources/ui/preferences.ui:21
+msgid "None"
+msgstr "Ninguno"
+
+#: data/resources/ui/preferences.ui:22
+msgid "End"
+msgstr "Fin"
+
+#: data/resources/ui/preferences.ui:23
+msgid "Start"
+msgstr "Inicio"
+
+#: data/resources/ui/preferences.ui:33
+msgid "Color Formats"
+msgstr "Formatos de color"
+
+#: data/resources/ui/preferences.ui:34
+msgid "Customize the visible formats and in which order they are displayed."
+msgstr "Personaliza los formatos visibles y el orden en que se muestran."
+
+#: data/resources/ui/preferences.ui:38
+msgctxt "Tooltip of the button to reset the format order"
+msgid "Reset Order"
+msgstr "Restablecer orden"
+
+#: data/resources/ui/shortcuts.ui:11
+msgctxt "shortcut window"
+msgid "General"
+msgstr "General"
+
+#: data/resources/ui/shortcuts.ui:14
+msgctxt "shortcut window"
+msgid "Show Preferences"
+msgstr "Mostrar preferencias"
+
+#: data/resources/ui/shortcuts.ui:20
+msgctxt "shortcut window"
+msgid "Show Shortcuts"
+msgstr "Mostrar accesos directos"
+
+#: data/resources/ui/shortcuts.ui:26
+msgctxt "shortcut window"
+msgid "Quit"
+msgstr "Salir"
+
+#: data/resources/ui/shortcuts.ui:34
+msgctxt "shortcut window"
+msgid "Color"
+msgstr "Color"
+
+#: data/resources/ui/shortcuts.ui:37
+msgctxt "shortcut window"
+msgid "Pick a new Color"
+msgstr "Elegir un nuevo color"
+
+#: data/resources/ui/shortcuts.ui:43
+msgctxt "shortcut window"
+msgid "Randomize the Color"
+msgstr "Aleatorizar el color"
+
+#: data/resources/ui/window.ui:6
+msgid "_Random Color"
+msgstr "_Color aleatorio"
+
+#: data/resources/ui/window.ui:12
+msgid "_Preferences"
+msgstr "_Preferencias"
+
+#: data/resources/ui/window.ui:16
+msgid "_Keyboard Shortcuts"
+msgstr "_Atajos del teclado"
+
+#: data/resources/ui/window.ui:20
+msgid "_About Eyedropper"
+msgstr "_Acerca de Eyedropper"
+
+#: data/resources/ui/window.ui:40
+msgctxt "Tooltip of the colorpicker button"
+msgid "Pick a Color"
+msgstr "Elige un color"
+
+#. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
+#: src/application.rs:167
+msgid "translator-credits"
+msgstr "Sergio Varela https://github.com/IngrownMink4"
+
+#: src/widgets/preferences.rs:214
+msgid "Move Up"
+msgstr "Subir de categoría"
+
+#: src/widgets/preferences.rs:216
+msgid "Move Down"
+msgstr "Bajar de categoría"
+
+#: src/widgets/preferences.rs:305
+msgid "Hex-Code"
+msgstr "Código hexadecimal"
+
+#: src/window.rs:460
+msgid "Failed to pick a color"
+msgstr "No se ha podido elegir un color"
+
+#. Translators: Do not replace the {}. These are used as placeholders for the copied values
+#: src/window.rs:567 src/window.rs:574
+msgid "Copied: “{}”"
+msgstr "\"{}\" copiado."
```

Comment:
Why did you use `"` instead of U+201C (Left double quotation mark) and U+201D (Right double quotation mark)?

Reply:
Oops, that's true. My intention was to use U+201C and U+201D but I got mixed up and put `"` instead. Anyway, in the Spanish version it is preferable to use U+00AB («) and U+00BB (»). But in eu.po, yeah, better with U+201C and U+201D.